bindings/nodejs/src/wallet.rs

```diff
@@ -122,10 +123,15 @@ pub fn call_wallet_method(mut cx: FunctionContext) -> JsResult<JsUndefined> {
                 Ok(())
             });
         } else {
-            panic!("Wallet got destroyed")
+            sender.send("Wallet got destroyed").unwrap();
```

Comment:
undocumented unwrap :shrug: (is it ensured that the receiver always be instantiated at this point?)

also ... you could probably just send a `()` and create the string "Wallet got destroyed" on the receiving end because it's probably the only thing that gets sent?

Reply:
Not sure about your first question, receiver is created the same time as the sender, so it always exists

Reply:
theoretically(!!) `receiver.recv()` could return `Err`, right? then `call_wallet_method` would exit/return with `Ok(cx.undefined())` deallocating `receiver`, which the spawn task would still be alive.

Reply:
ignore me... docs say  this can't happen as long as the sender is alive.

Reply:
oops. resolved too quickly ... the 2nd point still stands ... no need to send this `&str`.

Reply:
I changed that too?

Reply:
https://github.com/iotaledger/iota-sdk/blob/3954255ddd9cd4c7a69a89fb7ddebfe82bba82f6/bindings/nodejs/src/wallet.rs#L128

Reply:
rilly? then I resolve